### PR TITLE
NIST_SP800-53_v4.sarif and tool code

### PR DESCRIPTION
**Pull request by shaopeng-gh**

### CODE REVIEW

File: Tools/Taxonomy/Common/Options.cs [hunk 1]
```diff
@@ -38,8 +38,17 @@ internal class OwaspOptions : IOptions
         public string ReleaseDateUtc { get; set; }
     }
 
-    [Verb("generate-nistsp80053", false, HelpText = "Generate NIST SP800-63B Sarif file")]
-    internal class NistSP80053Options : IOptions
+    [Verb("generate-nistsp80053csv", false, HelpText = "Generate NIST SP800-63B Sarif file from Csv")]
+    internal class NistSP80053CsvOptions : IOptions
```
Comment (eddynaka) [inline Tools/Taxonomy/Common/Options.cs]: >NistSP80053CsvOptions [](http://example.com/codeflow?start=19&length=21)

you could merge all nist options in one, add two more arguments as format and type, for example.
and, internally, based on those parameters, you instantiate A or B

Comment (shaopeng-gh) [reply Tools/Taxonomy/Common/Options.cs]: sounds good, will work on it

Comment (shaopeng-gh) [reply Tools/Taxonomy/Common/Options.cs]: fixed

**Review by eddynaka — APPROVED** [review]
:shipit:

**Pull request merged**